File: Python/consecutive-characters.py
import math
class Solution:
    def maxPower(self, s: str) -> int:
        count=1
        maxcount=1
        for i in range(1,len(s)):
            if(s[i]==s[i-1]):
                count+=1
                maxcount=max(maxcount,count)
            else:
                count=1
        return maxcount

File: Java/Consecutive-characters.java
class Solution {
    public int maxPower(String s) {
        int count=1;
        int maxcount=1;
        for(int i=1;i<s.length();i++){
            if(s.charAt(i) == s.charAt(i-1)){
                count++;
                maxcount=Math.max(maxcount,count);
                }
            else{
                count=1;
                }
            }
            return maxcount;
            }
        }
